prow/cmd/phaino/local.go

```diff
@@ -316,6 +417,26 @@ func convertToLocal(ctx context.Context, log *logrus.Entry, pj prowapi.ProwJob,
 	return localArgs, nil
 }
 
+// envExists checks if the given env name exists in the env var list
+func envExists(envs []coreapi.EnvVar, envName string) bool {
+	for _, env := range envs {
+		if env.Name == envName {
+			return true
+		}
+	}
+	return false
+}
+
+// strExists checks if the given string exists in the string list
+func strExists(strs []string, s string) bool {
```

Comment:
consider sets.String for this, not sure if it is worth it here

Reply:
Done. Thanks.